jest.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
   ],
   roots: ['<rootDir>/packages'],
   transform: {
-    '^.+\\.[jt]sx?$': 'babel-jest'
+    '^.+\\.m?[jt]sx?$': 'babel-jest'
```

Comment:
what does this check?

Reply:
Makes it so that the transformer also picks up .mjs files, which are what the react-styles build process outputs for ESM.